test/e2e.test.js

```diff
@@ -36,12 +36,23 @@ test('Check that any-na linkedIn links are not selected', function(t) {
 })
 
 test('Check that the link of a pdf are correctly parsed', function(t) {
-  var keywords =  [ 'Abdi', 'Ahmed', 'Founders', 'and', 'Coders'];
+  var keywords =  [ 'Ines', 'Teles', 'Founders', 'Coders', 'pdf'];
```

Comment:
I just had the pdf in the keywords to get some random pdf links in the google results.

![pdflinks](https://cloud.githubusercontent.com/assets/6057298/10589861/9124269e-76a5-11e5-9472-32178b9c4088.png)